File: Execution/SmartBox/SmartBox/Models/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SmartBox.Models
{
    public class Location
    {
        public int ID { get; set; }
        public string Longitude { get; set; }
        public string Latitude { get; set; }
        public string Allitude { get; set; }
        public string Time { get; set; }
        public bool IsOpen { get; set; }

        public override string ToString()
        {
            return $"Long: {Longitude}, Lat: {Latitude}, time: {Time}, open: {IsOpen}";
        }
    }
}

File: Execution/App/app/src/main/java/sbd/smartbox/models/Location.java
package sbd.smartbox.models;

import android.support.annotation.Nullable;

public class Location {
    private int id;
    private String longitude;
    private String latitude;
    private String altitude;
    private String time;
    private Boolean isOpen;

    public Location(int id, String longitude, String latitude, String altitude, String time, Boolean isOpen) {
        this.id = id;
        this.longitude = longitude;
        this.latitude = latitude;
        this.altitude = altitude;
        this.time = time;
        this.isOpen = isOpen;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getLongitude() {
        return longitude;
    }

    public void setLongitude(String longitude) {
        this.longitude = longitude;
    }

    public String getLatitude() {
        return latitude;
    }

    public void setLatitude(String latitude) {
        this.latitude = latitude;
    }

    public String getAltitude() {
        return altitude;
    }

    public void setAltitude(String altitude) {
        this.altitude = altitude;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public boolean getIsOpen() {
        return isOpen;
    }

    public void setIsOpen(Boolean isOpen) {
        this.isOpen = isOpen;
    }
}
